scripts/pre-commit

```diff
@@ -18,6 +18,12 @@ if ! make -k lint; then
     failed=1
 fi
 
+# echo "📋 Type Checking, expect failures for now"
```

Comment:
Why it's commented out?

Reply:
I have some work in a separate branch to completely remove existing type-check complaints. That's more involved work across all of committime.

Until then, I don't want the pre-commit check to be spammed with errors-- that's how one gets error fatigue and starts ignoring it.

I can remove this if it's cleaner than commenting it out. But I think of it as a feature flag, y'know? (Could also put it behind an environment variable).